### PR TITLE
EASE-UI Updates

### PR DESCRIPTION
**Pull request by Whatthefoxsays**
**Description of the change**:
**Reason for the change**:
**Link to original source**:

Closes #

### CODE REVIEW

File: source/User_Guide/email_activity_beta.md [hunk 1]
```diff
@@ -21,33 +22,100 @@ The Email Activity feed allows you to view a snapshot of email-related activity
...
+    <td class="tg-031e">Processed</td>
+    <td class="tg-yw4l">Requests from your website, application, or mail client via SMTP Relay or the API that SendGrid processed.</td>
+  </tr>
+  <tr>
+    <td class="tg-031e">Clicks</td>
+    <td class="tg-yw4l">Whenever a recipient clicks one of the Click Tracked links in your email.</td>
+  </tr>
+  <tr>
+    <td class="tg-031e">Delivered</td>
+    <td class="tg-yw4l">An email that was delivered to a recipient.</td>
```
Comment (ksigler7) [inline source/User_Guide/email_activity_beta.md]: Passive. How about:

The accepted response generated by the recipients' mail server.

File: source/User_Guide/email_activity_beta.md [hunk 2]
```diff
@@ -21,33 +22,100 @@ The Email Activity feed allows you to view a snapshot of email-related activity
...
+  </tr>
+  <tr>
+    <td class="tg-yw4l">Opens</td>
+    <td class="tg-yw4l">Whenever an email is opened by a recipient.</td>
```
Comment (ksigler7) [inline source/User_Guide/email_activity_beta.md]: Passive. How about:

The response generated by a recipient opening an email.

File: source/User_Guide/email_activity_beta.md [hunk 3]
```diff
@@ -21,33 +22,100 @@ The Email Activity feed allows you to view a snapshot of email-related activity
...
+    <td class="tg-yw4l">Deferred</td>
+    <td class="tg-yw4l">The recipient mail server asked SendGrid to stop sending emails so fast.</td>
+  </tr>
+  <tr>
+    <td class="tg-yw4l">Drops</td>
+    <td class="tg-yw4l">SendGrid will drop an email when the contact on that email is in one of your suppression groups, the recipient email previously bounced, or that recipient has marked your email as spam.</td>
+  </tr>
+  <tr>
+    <td class="tg-yw4l">Bounces </td>
+    <td class="tg-yw4l">When an email is attempted to be delivered, but the recipient mail server rejects it.</td>
```
Comment (ksigler7) [inline source/User_Guide/email_activity_beta.md]: The receiving server could not or would not accept the message. If a recipient has previously unsubscribed from your emails, your attempt to send to them is bounced.

File: source/User_Guide/email_activity_beta.md [hunk 4]
```diff
@@ -21,33 +22,100 @@ The Email Activity feed allows you to view a snapshot of email-related activity
...
+    <td class="tg-yw4l">Blocks</td>
+    <td class="tg-yw4l">When your IP address has been blacklisted or blocked by an ISP or messaging organization. Blocks are less severe than bounces and do not result in permanent suppressions: subsequent sends to blocked email addresses are not automatically suppressed.</td>
+  </tr>
+  <tr>
+    <td class="tg-yw4l">Spam Reports</td>
+    <td class="tg-yw4l">Whenever a recipient marks your email as spam and their mail server tells us about it.</td>
+  </tr>
+  <tr>
+    <td class="tg-yw4l">Parse</td>
+    <td class="tg-yw4l">Whenever an incoming email is processed through the Inbound Parse Webhook.</td>
```
Comment (ksigler7) [inline source/User_Guide/email_activity_beta.md]: Triggered if you're using the Inbound Parse Webhook.

File: source/User_Guide/email_activity_beta.md [hunk 5]
```diff
@@ -21,33 +22,100 @@ The Email Activity feed allows you to view a snapshot of email-related activity
...
+  </tr>
+</table>
+
+{% anchor H2 %}
 Filtering email activity
 {% endanchor %}
 
-Events on the Activity feed are organized by recipient email address and can be filtered with our basic and advanced search capabilities. 
+Events on the Activity feed are organized by email and can be filtered with our basic and advanced search capabilities.
```
Comment (ksigler7) [inline source/User_Guide/email_activity_beta.md]: This has a passive in it, but I rewrote more than the passive for clarity:

The Email Activity feed lists each email sent. Click on each email to view the current list of triggered events for that email. Use our basic or advanced search to filter by email subject, recipients, or event types.

File: source/User_Guide/email_activity_beta.md [hunk 6]
```diff
@@ -21,33 +22,100 @@ The Email Activity feed allows you to view a snapshot of email-related activity
...
+Events on the Activity feed are organized by email and can be filtered with our basic and advanced search capabilities. 
+
 
-{% anchor h3 %}
+{% anchor H3 %}
 Basic search
 {% endanchor %}
 
-By default, the Activity feed allows you to search by recipient email address and event timeframe. The To email address field will filter results including approximate data, or fuzzy search. The date field will filter results that fit the precise date range specified.
+By default, the Activity Feed allows you to search by email address and event dates. The email address field filters with approximate, or fuzzy search. The date field pulls filters events that fit within the precies date range specified.
```
Comment (ksigler7) [inline source/User_Guide/email_activity_beta.md]: The email address field filters with an approximate, or fuzzy search. The date field pulls filters events that fit within the precise date range specified.

File: source/User_Guide/email_activity_beta.md [hunk 7]
```diff
@@ -65,34 +133,34 @@ In addition to viewing the email activity associated with your account by recipi
 
 *To use advanced search:*
 
-1. At the top of the Activity Feed page, click **Advanced** search, in the upper right of the search window.
+1. At the top of the Activity Feed page, click **Advanced** search. 
 1. In the field on the far left, select the criteria you wish to filter your event data by.
```
Comment (ksigler7) [inline source/User_Guide/email_activity_beta.md]: In the field on the far left, select the criteria to filter your event data.

File: source/User_Guide/email_activity_beta.md [hunk 8]
```diff
@@ -65,34 +133,34 @@ In addition to viewing the email activity associated with your account by recipi
...
-To add additional search criteria, click **Add a Filter**. 
+1. Once you have filled in all of the fields associated with that filter, click **Search**. 
+   To add additional search criteria, click **Add a Filter**. 
 
 {% info %}
 To further narrow your search results, select one of the following options:
 
-* Search emails by **any** of the following - This search is an expansive. It will return more results and will pull in all events that meet *one or more* of the criteria specified by the selected filters.
-* Search emails by **all** of the following - This search is limiting. It will return fewer results and will pull in events that meet *every* criteria specified by the selected filters. 
+* Search emails by **any** of the following - This search will return more results and will pull in all events that meet *one or more* of the criteria specified by the selected filters.
```
Comment (ksigler7) [inline source/User_Guide/email_activity_beta.md]: Grammerly says this should be "criterion"

File: source/User_Guide/email_activity_beta.md [hunk 9]
```diff
@@ -65,34 +133,34 @@ In addition to viewing the email activity associated with your account by recipi
...
 Exporting a CSV
 {% endanchor %}
 
 *To export a CSV:*
 
-1. In the top right corner of the Activity feed, click **Export CSV**. 
-   An email is sent to the email address associated with your SendGrid account *or* the address associated with your Marketing Campaigns feature found under **Notifications**.
-1. Open the email and then click **Download**.
+1. In the top right corner of the activity feed, click **Export CSV**. 
+   An email is sent to the email address associated with your SendGrid account.
```
Comment (ksigler7) [inline source/User_Guide/email_activity_beta.md]: Passive. How about:
This triggers an email to the email address associated with your SendGrid account.

File: source/User_Guide/email_activity_beta.md [hunk 10]
```diff
@@ -21,33 +22,100 @@ The Email Activity feed allows you to view a snapshot of email-related activity
...
+{% anchor H2 %}
+Types of email activity data
+{% endanchor %}
+
+The Email Activity feed allows you to view specific information about messages sent from your account. 
+
+
+<table class="table" style="table-layout:fixed">
+  <tr>
+  <th class="tg-031e">Event</th>
```
Comment (ksigler7) [inline source/User_Guide/email_activity_beta.md]: also, I just realized that you should delete all of these random classes. That's generated automatically by the table generator website, but you can turn it off. Click the box for "Do not generate CSS".

File: source/User_Guide/email_activity_beta.md [hunk 11]
```diff
@@ -10,9 +10,10 @@ seo:
   description:
 ---
 
-The Email Activity feed allows you to view a snapshot of email-related activity associated with your account and any subusers. Our latest enhancements to Email Activity give you increased visibility into the email delivery flow to help expedite the troubleshooting process.
+The Email Activity feed allows you to view a snapshot of email-related activity associated with your account and any subusers. Enhanced email activity and storage gives you increased visibility into the email delivery flow to help expedite the troubleshooting process.
```
Comment (lweinig) [inline source/User_Guide/email_activity_beta.md]: maybe 'email delivery flow of each email message to expedite'... Would be great to highlight the fact that the new email activity feed shows events by email message vs. old email activity's log of disparate events that could be tied to the same message.

File: source/User_Guide/email_activity_beta.md [hunk 12]
```diff
@@ -21,33 +22,100 @@ The Email Activity feed allows you to view a snapshot of email-related activity
 
 With the Email Activity feed you can:
 
-* Troubleshoot email delivery issues faster with comprehensive and sequential event data.
-* Pinpoint specific emails easily with advanced search and filter options -- including subject line and email metadata.
+* Troubleshoot email delivery issues faster with comprehensive and sequential event data by email.
```
Comment (lweinig) [inline source/User_Guide/email_activity_beta.md]: by email message (just so people don't confuse it with email address)

File: source/User_Guide/email_activity_beta.md [hunk 13]
```diff
@@ -21,33 +22,100 @@ The Email Activity feed allows you to view a snapshot of email-related activity
...
+  <th class="tg-031e">Event</th>
+    <th class="tg-yw4l">Description</th>
+    </tr>
+    <tr>
+    <td class="tg-031e">Processed</td>
+    <td class="tg-yw4l">Requests from your website, application, or mail client via SMTP Relay or the API that SendGrid processed.</td>
+  </tr>
+  <tr>
+    <td class="tg-031e">Clicks</td>
+    <td class="tg-yw4l">Whenever a recipient clicks one of the Click Tracked links in your email.</td>
```
Comment (lweinig) [inline source/User_Guide/email_activity_beta.md]: In the Email History we display the date, time and the URL for the link that was clicked

File: source/User_Guide/email_activity_beta.md [hunk 14]
```diff
@@ -65,34 +133,34 @@ In addition to viewing the email activity associated with your account by recipi
...
-{% anchor h2 %}
+{% anchor H2 %}
 Viewing Event History
 {% endanchor %}
 
 *To view event history:*
 
 1. Navigate to the event you wish to view and click on the entry. 
-1. When you have finished reviewing the event data, click **close**. 
+1. When you have finished reviewing the data, click **close**.
```
Comment (lweinig) [inline source/User_Guide/email_activity_beta.md]: May want to mention they can click to expand sections for more details? This is where they'd find the clicked URL

File: source/User_Guide/email_activity_beta.md (hunk 11 above)
Comment (lweinig) [reply source/User_Guide/email_activity_beta.md]: Also are we sure subuser information is shown here? I'll confirm with Scott K.

Comment (lweinig) [reply source/User_Guide/email_activity_beta.md]: You are right, it is.

**Comment by thinkingserious** [comment]
Hello @Whatthefoxsays,<br /><br/>Thanks again for the PR!<br />                                              <br />We appreciate your contribution and look forward to                                              continued collaboration. Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**